src/averageAges.js

```diff
@@ -37,7 +46,17 @@ function calculateMenAverageAge(people, century) {
  * @return {number}
  */
 function calculateWomenAverageAge(people, withChildren) {
-  // write code here
+  let women = people.filter(x => x.sex === 'f');
+
+  if (withChildren) {
```

Comment:
> replace `if ()` statement with &&, || or ?:

use `withChildren` inside the filter callback:
```
const women = people.filter(person =>
  withChildren
   ? person.sex === 'f'
   : person.sex === 'f' && people.some(child => person.name === child.mother)
);
```
the same in the `calculateMenAverageAge`

Reply:
Not fixed. If something is not clear, ask in the chat. For now, fix the code as recommended